api/client/src/docs/Failure.md

```diff
@@ -0,0 +1,13 @@
+# Failure
+
+
+## Properties
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+**failure_code** | **str** | Failure code when the cluster stack is in CREATE_FAILED status. | [optional] 
+**failure_reason** | **str** | Failure reason when the cluster stack is in CREATE_FAILED status. | [optional] 
+**any string name** | **bool, date, datetime, dict, float, int, list, str, none_type** | any string name can be used but the value must be the correct type | [optional]
```

Comment:
[Nitpick] Should the description be capitalized?

Reply:
This is auto generated